tensorflow/lite/kernels/activations.cc

```diff
@@ -1113,22 +1093,44 @@ TfLiteStatus LeakyReluEval(TfLiteContext* context, TfLiteNode* node) {
   const LeakyReluOpData* data =
       reinterpret_cast<LeakyReluOpData*>(node->user_data);
 
-  LeakyReluParams op_params;
-  op_params.alpha = params->alpha;
   switch (input->type) {
     case kTfLiteFloat32: {
+      LeakyReluParams op_params{params->alpha};
       optimized_ops::LeakyRelu(
           op_params, GetTensorShape(input), GetTensorData<float>(input),
           GetTensorShape(output), GetTensorData<float>(output));
       return kTfLiteOk;
     } break;
     case kTfLiteUInt8: {
-      QLeakyRelu<uint8_t>(input, output, params->alpha, data);
+      LeakyReluParams op_params{0.,
```

Comment:
nit: we don't usually use default initializer for struct if the struct has many fields.

Explicitly populating the fields is more readable. thanks!

Reply:
Moved op_params initialization outside of switch statement and populated fields explicitly. Thank you for tips.